### ISSUE TITLE
is it able to work with Octane?

### ISSUE THREAD

**Issue by arivelli** [gecche/laravel-multidomain#55]
Hey Folks,

I was able to put my project (which use laravel-multidomain) to work with octane (swoole), with no additional effort it is able to run cli commands.
But when I try from web looks like only use the original .env file no matter the domain I use. Any suggestion to solve it?

Thank in advance

**Comment by gecche** [comment]
Hi,

sorry for the delay... I'm very very constrained in this period...

However, I never used Octane, so I'm not able to properly answer you... 
Anyway, since the CLI is working, maybe Octane does some customization in the `App\Http\Kernel` class or even in the main  `Illuminate\Foundation\Application` class

But I don't have the code, so I'm only guessing

Cheers

Giacomo

**Comment by arivelli** [comment]
any news here?

**Comment by gecche** [comment]
nope… I’m sorry but I don’t use Octane so I’m not be able to help… I left the issue open but it seems nobody could help 

cheers

giacomo

**Comment by raulamoretti** [comment]
I will test Octane + Swoole with multidomain! If somebody test before, call here!

**Comment by negartarh** [comment]
> I will test Octane + Swoole with multidomain! If somebody test before, call here!

Hi
What was the result of your test?

**Comment by valsha** [comment]
> I will test Octane + Swoole with multidomain! If somebody test before, call here!
Hi, any news? Thanks.

**Comment by raulamoretti** [comment]
Hey guys!

In my initial tets it dont work. Only get original .env.
I will commit in github my test.

Thanks!

**Comment by valsha** [comment]
@gecche any chance to add support Octane?  Thank you.

**Comment by fpolli** [comment]
I think Octane support would require a massive change in architecture of the package. Octane splits off part of execution to run once and stay in memory. This package is based on the normal execution where the entire execution flow is run per request. I plan to use Octane with FrankenPHP so maybe I will fork the repo here and develop it for that paradigm. I need it anyway. It would be great if a more experienced coder were to do it, but...
